src/components/RoomView/RoomView.tsx

```diff
@@ -111,34 +111,23 @@ const RoomView = () => {
         }
 
```

Comment:
we only need to update firebase here, because when the game status is complete, redux will be triggered (see line 127)

since the game is considered 'complete' and a different winner/loser is set when an assassin is hit, having 0 remaining cards is not the 'only' way that a team can win. so i think it might make more sense to do it this way!